boreas/boreas_io.c

```diff
@@ -474,6 +474,12 @@ get_alive_test_methods (alive_test_t *alive_test)
     {
       *alive_test = atoi (alive_test_pref_as_str);
     }
+
+  // If consider_alive method is set, unset the other methods
```

Comment:
Not sure if this is the cleanest solution. Feels a bit like hidden logic to place it here.
Other solution would be to handle the ALIVE_TEST_CONSIDER_ALIVE first in the ping functions and just abort after or not start the others.
We can discuss tomorrow.